### PR TITLE
#1-4Task. AleksKovalenko

### PR DESCRIPTION
**Pull request by alextrapper**
Successful build for "Tasks 1 - 4" branch is [here](https://travis-ci.org/github/alextrapper/js-assignments/builds/705671252)

### CODE REVIEW

File: task/01-strings-tasks.js [hunk 1]
```diff
@@ -83,9 +84,8 @@ function extractNameFromTemplate(value) {
  *   'John Doe'  => 'J'
  *   'cat'       => 'c'
  */
-function getFirstChar(value) {
-    throw new Error('Not implemented');
-}
+let getFirstChar = (value) => value.match(/^./g)[0];
```
Comment (FedorovD) [inline task/01-strings-tasks.js:87]: You shouldn't change the definition of functions, just do all changes inside already exists functions

File: task/01-strings-tasks.js [hunk 2]
```diff
@@ -98,9 +98,7 @@ function getFirstChar(value) {
  *   'cat'              => 'cat'
  *   '\tHello, World! ' => 'Hello, World!'
  */
-function removeLeadingAndTrailingWhitespaces(value) {
-    throw new Error('Not implemented');
-}
+let removeLeadingAndTrailingWhitespaces = (value) => value.trim();
```
Comment (FedorovD) [inline task/01-strings-tasks.js:101]: same

**Comment by FedorovD** [comment]
Please, return all the functions to their initial view and write solutions inside them.
**I'll mark the PR as "Draft", please click "ready for review" when it will be finished. Thank you!**

**Comment by alextrapper** [comment]
All functions declarations returned to their initial view, sorry

**Comment by alextrapper** [comment]
New build https://travis-ci.org/github/alextrapper/js-assignments/builds/707616185

**Comment by FedorovD** [comment]
Closing the PR as accepted! Thanks!

**Pull request closed without merging**